doc/api/tty.md

```diff
@@ -106,6 +106,60 @@ added: v0.7.7
 A `number` specifying the number of columns the TTY currently has. This property
 is updated whenever the `'resize'` event is emitted.
 
+### writeStream.cursorTo(x, y)
+<!-- YAML
+added: v0.7.7
+-->
+
+* `x` {number}
+* `y` {number}
+
+`writeStream.cursorTo` moves this `WriteStream`'s cursor to the specified
```

Comment:
`` `writeStream.cursorTo` `` -> `` `writeStream.cursorTo()` `` as per our [`doc/STYLE_GUIDE.md`](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md). The same for the method mentions below.